core/src/main/java/nl/nn/adapterframework/util/AppConstants.java

```diff
@@ -240,6 +250,9 @@ private synchronized void load(ClassLoader classLoader, String filename, boolean
 	}
 
 	private synchronized void load(ClassLoader classLoader, String filename, String suffix, boolean loadAdditionalPropertiesFiles) {
+		if(StringUtils.isEmpty(filename))
+			throw new IllegalStateException("file to load properties from cannot be null");
```

Comment:
Ik doe altijd snorhaken zetten om een then clause, als hij op de volgende regel staat.

Reply:
Is goed, ik zal er een accolade om heen zetten!